update.sh

```diff
@@ -3,13 +3,15 @@ set -e
 
 declare -A gpgKeys
 gpgKeys=(
+	[7.0]='1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763'
```

Comment:
@yosifkit no, If I just add that line, when checking the versions on that packagesUrl 7.0 is not there yet so we don't have the gpg neither the fullVersion